homeassistant/components/remote/__init__.py

```diff
@@ -16,7 +16,6 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.config_validation import (  # noqa: F401
     PLATFORM_SCHEMA,
-    PLATFORM_SCHEMA_BASE,
```

Comment:
Interesting, why did neither our githook nor CI catch this as unused.

Reply:
Base integrations that provide platform support should have a `PLATFORM_SCHEMA_BASE`. So this should be kept.

Reply:
https://github.com/home-assistant/core/blob/940d28960b58406b8d7b03e70509e6a515a48be4/homeassistant/config.py#L811-L813